sharing/README.md

```diff
@@ -0,0 +1,208 @@
+# Aim/Objective:
+
+The purpose of the ODM is to support wastewater-based surveillance and epidemiology by facilitating the collection, standardization, and transparency of data by providing a more harmonized ODM data format to share data between different data curators and data repositories.
+
+The ODM supports data sharing in two ways:
+
+1. **Data sharing schema** - The ODM will have a schema that describes what data can be shared with one or more partners or users.
+2. **Data filter based on the data sharing schema** - The ODM will support an open source method for filtering data tables using the data sharing schema.
+
+The data sharing schema will be a csv file (sharing.csv) where each row in the file corresponds to one variable in a ODM table. Attributes in the row describe who the data is shared with and what data is excluded. See below for an example.
+
+The data filter is a Python module (or function) that excludes data based on a data sharing schema. The function accepts an ODM data tables and sharing schema. The function excludes (filters) data variables and rows according to the schema rules. The function then returns a data table with only the data that is to be shared. This new, returned data is ready to be shared and used with a partner.
+
+# Features
+
+High level features include:
+
+- The data custodian should be able to define all the sharing rules in a CSV file (`sharing.csv`). A standard schema for defining the rules will be developed.
+- The schema should allow a data custodian to define the partner (organization or person) that each rule pertains to. For example, a certain rule may be applicable only to the Public Health Agency of Canada (PHAC) while another rule may be applicable to not only the PHAC but also to Ottawa Public Health.
+- The schema should allow data custodians to define rules that applies to rows or to columns. For example, a rule can be made to exclude all the rows from the `Sample` table or to exclude the `type` column from the `Sample` table.
+- Rules can be made within the context of an entire table, to a column that may be present in more than one table or to a column specific to a table.
+- The rules may only be exclusive. For example, rules can be defined to remove rows but not to include them.
+- Rules can be combined to form more powerful conditions using logical operators. For example, exclude all rows with `contactEmail` equal to "john.doe@email.com" AND `contactName` equal to "John Doe". Current supported logical operators are `AND` and `OR`.
```

Comment:
If we can only exclude data using the filters, then I think we should also support the `NOT` operator. That way, a curator could make a rule to exclude, say, all sites except a few specific ones which have been cleared for sharing. Otherwise, the curator will be forced to keep a list of all entries in his table to achieve the same effect.

Reply:
Nevermind, I just saw on line 132 that it `NOT`is supported - then we can add it here

Reply:
I'll add it to this line